packages/web-extension/test/walletManager/WalletRepository.test.ts

```diff
@@ -290,6 +332,33 @@ describe('WalletRepository', () => {
       ]);
     });
 
+    it('does not update 1852 account metadata when updating 1854 account', async () => {
+      const newAccount = createAccount(0, 1, KeyPurpose.MULTI_SIG);
```

Comment:
Let's make sure it's the same accountIndex. Same for `does not remove 1852 account when removing 1854 account`
```suggestion
      const storedAccount = storedLedgerWallet.accounts[0];
      const newAccount = createAccount(1, storedAccount.accountIndex, KeyPurpose.MULTI_SIG);
```

Reply:
I think you forgot to fix this https://github.com/input-output-hk/cardano-js-sdk/pull/1317#discussion_r1631182597